src/main/java/com/xebialabs/restito/semantics/Condition.java

```diff
@@ -203,6 +205,15 @@ public boolean apply(Call input) {
         });
     }
 
+    public static Condition withPostBodyContainingJsonPath(final String pattern, final String value) {
```

Comment:
Could you please add javadoc for this method? Also what happens if:
* Body does not contain valid json;
* Pattern is invalid;
* Pattern doesn't match anything?